### PR TITLE
week2

### PR DESCRIPTION
**Pull request by hemagunti**

### CODE REVIEW

File: week2/week2.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    def __init__(self, name, country):
+        self.name = name
+        self.country = country
+
+    def jet_details(self):
+        print(f'{self.name} {self.country}')
+
+
+jetInventory1 = JetInventory("AirIndia", "India")
+jetInventory1.jet_details()
```
Comment (Rassibassi) [inline week2/week2.py]: Above for `num_list` you are using `snake_case`, but now for `jetInventory1` you are using `camelCase`, be consistence, use either one, but then always

File: week2/week2.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+is_key_present(5)
+is_key_present(9)
+
+
+# 6.Identify the even and odd numbers.
+
+def show_numbers(limit):
+    num = int(input("Enter a number: "))
+    mod = num % 2
+    if mod > 0:
```
Comment (Rassibassi) [inline week2/week2.py]: I would use `!=` here instead of `>`

**Review by Rassibassi — COMMENTED** [review]
Good job :)

File: week2/week2.py (hunk 2 above)
Comment (hemagunti) [reply week2/week2.py]: Thank you for the feedback :)

**Pull request merged**